carmajava/rsumetering/src/main/java/gov/dot/fhwa/saxton/carma/rsumetering/CommandingState.java

```diff
@@ -96,6 +105,16 @@ public void onMobilityOperationMessage(MobilityOperation msg) {
     // Simply updating the command speed to the platoon speed may be enough to make this work
     // If it is not more complex logic can be added
 
+    //perform sanity check on incoming params
+    if (meterDist < MIN_METER_DIST  ||  meterDist > MAX_METER_DIST  ||
+        mergeDist < MIN_MERGE_DIST  ||  mergeDist > MAX_MERGE_DIST  ||
+        speed < MIN_SPEED           ||  speed > MAX_SPEED  ||
+        lane < MIN_LANE             ||  lane > MAX_LANE) {
+      log.warn("Received operation message with suspect strategy variables. meterDist = " + meterDist +
+                ", mergeDist = " + mergeDist + ", speed = " + speed + ", lane = " + lane);
+      return;
```

Comment:
This is the commanding state of the RSU so it is actively controlling the vehicle. We need to send a NACK message here so that the commanded vehicle will return to cruising control.